pallets/nfts/src/default_weights.rs

```diff
@@ -1,39 +1,135 @@
-use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
+/*
+ * This file is part of the Nodle Chain distributed at https://github.com/NodleCode/chain
```

Comment:
Is this correct?

Reply:
No :-)

Reply:
Oh crap, that comes from the template I used

Reply:
I had added a weights template for when we generate them

Reply:
Should be fixed :)

Reply:
Thanx for the quick fix :heart: